Ron/mssqlcli refactor

MrMeemus opened the pull request:
- Moved MssqlCli into it's own file (__init__.py).
- Created a mssqlutils module that has the static methods used by MssqlCli.
- Removed unused static methods such as is_mutating_cmd, etc.
- Renamed mssqlutils that was used by tests to mssqltestutils.
- Moved telemetry session server log to main.py as MssqlCli should not be aware of a telemetry object.

pensivebrian commented on mssqlcli/__init__.py:
```diff
@@ -1 +1,638 @@
+
+
+class MssqlCli(object):
+
```
Do we have to put this in __init__.py? I'd rather see it in mssqlcli.py

abhisheksinha89 replied:
Yea putting stuff in init.py is misleading IMO. Agree with Brian

---
In reply to: [171445777](https://github.com/dbcli/mssql-cli/pull/173#discussion_r171445777) [](ancestors = 171445777)

pensivebrian commented on mssqlcli/mssqlutils.py:
```diff
@@ -0,0 +1,110 @@
+
+
+def has_change_db_cmd(query):
+    """Determines if the statement is a database switch such as 'use' or '\\c'
```
My preference... if a method is only used in one file, define the method that file. In this case, has_changed_db_cmd is only used by MssqlCli, so I think it should be there.

pensivebrian replied:
This may apply to other methods in this file.

---
In reply to: [171446837](https://github.com/dbcli/mssql-cli/pull/173#discussion_r171446837) [](ancestors = 171446837)

MrMeemus commented:
Refactored based on comments. I moved MssqlCli into mssql_cli.py because when I named it mssqlcli there was a issue with telelemtry upload where it would try to resolve mssqlcli.config from the module rather than the package level. We can refactor this later, it's just not as a high priority for now.
